tox.ini

```diff
@@ -1,17 +1,12 @@
 [tox]
 envlist =
-    py{33}-django18
-    py{27,34,35}-django{18,19,110,111}
-    py{36}-django{111}
-    py{34,35,36}-django2
+    py{27,34,35,36}-django111
+    py{34,35,36}-django20
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
     django111: Django>=1.11,<2.0
-    django2: Django>=2.0,<2.1
+    django20: Django>=2.0,<2.10
```

Comment:
Next release is 2.1, not 2.10. Revert this line.

Reply:
Oops, fixed